mw-webapp/src/component/footer/Footer.module.scss

```diff
@@ -1,36 +1,44 @@
 @import "src/styles/variables";
 
-$userNameFontSize: 16px;
+$logoHeight: 55px;
+$privacyPaddingTop: 10px;
+$privacyPaddingBottom: 12px;
```

Comment:
Is it possible to use 10px? Please ask designers

Reply:
Designer decided to apply 8px for padding top and bottom